client/src/remote/index.js

```diff
@@ -24,6 +24,8 @@ const {
   ipcRenderer
 } = window.getAppPreload();
 
+delete window.getAppPreload;
```

Comment:
Maybe a comment could be helpful here? I'll likely forget why we did this in a few weeks from now. If it is important, but looks like a hack, it shall be documented I believe.